src/client/client4.js

```diff
@@ -66,6 +70,19 @@ export default class Client4 {
         this.token = token;
     }
 
+    setCSRFFromCookie() {
```

Comment:
As long as we're properly checking that the cookie exists before doing this, I think it's fine to have here since other people may need this for their own browser-based integrations as well

Reply:
Not having a particular opinion here, reason why it was placed here was to parse the cookie directly from `loginById` - If you think we should implement a method ala `setCSRF` and call that in the webapp after the redux login method was called, that sounds good to me as well. Happy about any proposals. Thoughts @enahum @hmhealey @crspeller ?

Reply:
I’m 0/5 after HH comment